apps/store/src/components/Perils/Perils.tsx

```diff
@@ -1,15 +1,47 @@
 import styled from '@emotion/styled'
-import React, { useState, useCallback } from 'react'
+import React, { useState, useCallback, ReactNode } from 'react'
+import { mq, useBreakpoint } from 'ui'
 import * as Accordion from '@/components/Accordion/Accordion'
 import { SpaceFlex } from '@/components/SpaceFlex/SpaceFlex'
 import { CoverageList } from './CoverageList'
 import { Peril } from './Perils.types'
 
+const MAX_COLS = 4
+
 type Props = {
   items: Array<Peril>
 }
 
 export const Perils = ({ items }: Props) => {
+  const matchesMdScreen = useBreakpoint('md')
+
+  if (matchesMdScreen) {
+    const numberOfItemsPerColumn = Math.ceil(items.length / MAX_COLS)
+    const accordions: Array<ReactNode> = []
+    for (let i = 0; i < items.length; i += numberOfItemsPerColumn) {
+      const perils = items.slice(i, i + numberOfItemsPerColumn)
+      accordions.push(<PerilsAccordion key={i} perils={perils} />)
+    }
+
+    return <PerilsAccordionGrid fixedCols={items.length < 4}>{accordions}</PerilsAccordionGrid>
+  }
+
+  return (
+    <PerilsAccordionGrid>
+      <PerilsAccordion perils={items} />
+    </PerilsAccordionGrid>
+  )
+}
+
+const PerilsAccordionGrid = styled.div(({ fixedCols = false }: { fixedCols?: boolean }) => ({
+  display: 'grid',
+  [mq.md]: {
+    gridTemplateColumns: `repeat(auto-fit, ${fixedCols ? '20.75rem' : 'minmax(20.75rem, 1fr)'})`,
+    gap: 16,
```

Comment:
Slightly adjust the row-gap for md breakpoints :) 
<img width="1006" alt="Screenshot 2022-12-19 at 11 47 23" src="https://user-images.githubusercontent.com/6661511/208409093-83375e4d-5999-4f08-bf7c-885fe7d3b1f1.png">
```suggestion
    gap: 0.25rem 1rem,
```

Reply:
Gotcha! Yeah omit this if this is what you decided with Zak and co 🙌